resource-watch/resource-watch
# remove add search from desktop header search

## Overview
Remove add search logic from quick search.
Use a shared search component for both the search page & the header search.
Add arrow up/down functionality for selecting a search result.

## Testing instructions
1. Go to `/search` should work like before, arrow keys functionality added here also for result selection.
2. Search from header should work from any page. 
3. Use arrow up/down + enter when focused on the search input, to select a search result and open its corresponding url. 
4. Make sure search results are un-mounting when exiting the search page + closing the header quick search. 

## Pivotal task
https://www.pivotaltracker.com/story/show/155629600
